config/cp4waiops-custom/templates/xsmall-configmap.yaml

```diff
@@ -0,0 +1,360 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app.kubernetes.io/instance: aiops
+    app.kubernetes.io/managed-by: ibm-aiops-orchestrator
+    app.kubernetes.io/name: custom-sized-profiles
+    app.kubernetes.io/part-of: ibm-aiops 
```

Comment:
Are these labels required or that's just something tested using some arbitrary values, e.g.: 
- I see `app.kubernetes.io/name` is `custom-sized-profiles`, instead of the ConfigMap name, that is `aiops-custom-size-profile`.
- For `app.kubernetes.io/instance`, assume usually it equals to the AIOps Installation CR name, which can be any name given by end users, but in this case, it seems to be hard-coded.
-

Reply:
In fact, those labels can be removed and I will remove it later.